components/tooltip/src/Tooltip.js

```diff
@@ -10,12 +10,12 @@ import { TooltipBubble } from './TooltipBubble';
 // ==============================
 // Overwrite component
 // ==============================
-const Wrapper = forwardRef((props, ref) => <span ref={ref} {...props} />);
+const Wrapper = forwardRef(({ tag: Tag, ...props }, ref) => <Tag ref={ref} {...props} />);
 
 // ==============================
 // Component
 // ==============================
-export const Tooltip = ({ text, ...props }) => {
+export const Tooltip = ({ tag, text, ...props }) => {
```

Comment:
Same as above we need a test case for this change.

Reply:
Have added a test case https://github.com/WestpacGEL/GEL/pull/208/commits/9fcc2d7873c5243834a144aa767c1fd4d563d480, but it's throwing a wobbly. Mind looking in to that @jaortiz